allauth/socialaccount/helpers.py

```diff
@@ -160,8 +160,6 @@ def _social_login_redirect(request, sociallogin):
 
 
 def _complete_social_login(request, sociallogin):
-    if request.user.is_authenticated:
-        get_account_adapter(request).logout(request)
```

Comment:
This would never allow us to have multiple sessions. I can't think of anything bad happening by removing these two lines.